feat: added session support for delete method

talhatanveer333 opened the pull request:
This introduces session support for the delete method. Now we can pass session in option param and delete method would be the part of that session

talhatanveer333 committed:
feat: added session support for delete method

talhatanveer333 committed:
fix: typo fix

talhatanveer333 committed:
fix: code refactors

talhatanveer333 committed:
fix: reverting package name

dsanel commented:
@talhatanveer333 this looks nice. Can you please add tests and 1-2 examples into [README.md](https://github.com/dsanel/mongoose-delete/blob/master/README.md).

talhatanveer333 commented:
@dsanel Sure bro

talhatanveer333 committed:
Merge branch 'master' into feat/session-support-for-delete

talhatanveer333 committed:
fix: added test cases for session support

talhatanveer333 committed:
fix: added REAMDE for session support

talhatanveer333 commented:
@dsanel Bro I have added 3 test cases and added a package to support db transactions on tests. Also REAME.md file is updated as well. Please approve and merge

talhatanveer333 commented:
@dsanel Have you reviewed the PR?

dsanel commented:
@talhatanveer333 currently I am supper bussy, will try to check this week

talhatanveer333 commented:
@dsanel This is a reminder

talhatanveer333 commented:
@dsanel Any progress on this bro?

bshahwani commented:
Merging this would help so we can avoid using internal fork.

talhatanveer333 commented:
@dsanel This is a reminder

talhatanveer333 commented:
@dsanel Any progress on this?

talhatanveer333 commented:
@dsanel Merging this would add more users instead of internal forks
